ocaml/xapi/certificates.ml

```diff
@@ -24,33 +20,37 @@ open Client
 module D=Debug.Make(struct let name="certificates" end)
 open D
 
+type t_trusted = Certificate | CRL
```

Comment:
i don't know what a CRL is, but wikipedia says:

> '[a CRL] is "a list of digital certificates that have been revoked by the issuing certificate authority (CA) before their >scheduled expiration date and should no longer be trusted"'

so i'm confused as to why it is part of `t_trusted`

Reply:
is it the list that we trust, rather than the items on the list?

Reply:
The type represents certificates that are trusted by the pool to validate the other party on TLS connections, this includes the list of invalid certs, because the software "trusts" the list to be correct and true.

This is in juxtaposition of the server certificate which is not used to validate other certificates on connection, but is shown to the other party.